authentication/wildfly/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2022 Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wildfly.security.jakartaee</groupId>
+    <artifactId>authentication-vi</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>minimal-pom</name>
+    <url>http://maven.apache.org</url>
+
+    <properties>
+        <!--  Versions -->
+        <version.org.jboss.galleon>4.2.8.Final</version.org.jboss.galleon>
+        <version.org.jboss.logging>3.4.3.Final</version.org.jboss.logging>
+        <version.org.jboss.remoting-jmx>3.0.4.Final</version.org.jboss.remoting-jmx>
+        <version.org.wildfly>27.0.0.Alpha2-SNAPSHOT</version.org.wildfly>
```

Comment:
I understand why we're doing this now, but I don't know that we should merge it with a SNAPSHOT version.

Reply:
For now I would say this project is very much WIP so I think we are better just merging and evolving, especially so we can at least get some CI running.

However once we release a version of WildFly which is certified I think we should make sure all runners are updated to that version and create a corresponding tag of this project,